content/response_integrations/third_party/torq/pyproject.toml

```
@@ -0,0 +1,24 @@
+[project]
+name = "Torq"
+version = "1.0.0"
```

Version must be float

```suggestion
version = "1.0"
```